src/app/routes/contact.tsx

```diff
@@ -129,11 +129,16 @@ export default function Contact() {
 
     useEffect(() => {
         const savedMessage = localStorage.getItem('contact-message');
-        const savedAnonymousOption = localStorage.getItem('anonymous-option');
-        const savedContactOption = localStorage.getItem('contact-option');
         if (savedMessage) setMessage(savedMessage);
+
+        const savedAnonymousOption = localStorage.getItem('anonymous-option');
         if (savedAnonymousOption) setAnonymousOption(savedAnonymousOption);
+
+        const savedContactOption = localStorage.getItem('contact-option');
         if (savedContactOption) setContactOption(savedContactOption);
+
+        let savedEmail = localStorage.getItem('contact-email');
```

Comment:
Btw, is there a reason why all the others are `const` but this one not?

Reply:
I changed it to let for some debugging purposes, and then forgot to change it back to const :facepalm: